crates/autopilot/src/driver_api.rs

```diff
@@ -27,33 +27,55 @@ impl Driver {
     }
 
     pub async fn solve(&self, request: &solve::Request) -> Result<solve::Response> {
-        self.request_response("solve", request).await
+        self.request_response(&["solve"], Some(request)).await
     }
 
-    pub async fn execute(&self, request: &execute::Request) -> Result<execute::Response> {
-        self.request_response("execute", request).await
+    pub async fn execute(
+        &self,
+        solution_id: &str,
+        _request: &execute::Request,
+    ) -> Result<execute::Response> {
+        // TODO: should be execute
+        self.request_response(&["settle", solution_id], Option::<&()>::None)
+            .await
```

Comment:
This is a mismatch of the API specified in openapi and what is implemented in the driver. The driver expects no body because it passes solution_id in the URL.